src/observer.ts

```diff
@@ -6,16 +6,21 @@ export interface IObserverOptions {
     readonly forwardRef?: boolean
 }
 
+export interface IMobXReactObserver {
+    isMobXReactObserver: true
```

Comment:
From what I gather it's just a hack for testing purposes? Do we really need to pollute each component with it?

Reply:
No, just to make external reflection by others possible. But without devtools this becomes less relevant indeed. I'm fine to drop it until, at least until somebody needs it again :).

Reply:
Dropped